### PR TITLE
Implement rollback

### PR DESCRIPTION
**Pull request by pe1uca**
Implementation of rollback functionality to run "down" queries at will, with the option to stop at a given `target`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,6 +86,7 @@
     "lint-staged": "12.4.0",
     "pre-commit": "1.2.2",
     "prettier-standard": "16.4.1",
+    "shx": "^0.3.4",
```
Comment (theogravity) [inline package.json:89]: Thanks for the contrib, but I will not approve this PR with this package.  `cp` and `rm` is a pretty common command across unix distros and the flags used should be common as well.

**Comment by theogravity** [comment]
As you build this out, please also create unit tests as well that tests this functionality, thanks.

File: package.json (hunk 1 above)
Comment (pe1uca) [reply package.json:89]: Those commands don't exist in windows, so I added the `shx` to make the build cross platform.  

I'll remove the package from the PR and just have it in my local to be able to run them.  
In this case I'd recommend adding this as an advice for developers working in windows.

**Comment by pe1uca** [comment]
> As you build this out, please also create unit tests as well that tests this functionality, thanks.  

Yes, that's why I created this PR as a draft, thanks for the initial review.  
I'll create the tests.

**Comment by theogravity** [comment]
Going to close as it's been months without any updates.

**Pull request closed without merging**